provision/acc_provision/templates/aci-containers.yaml

```diff
@@ -17,7 +17,11 @@ metadata:
 ---
 {% endif %}
 {% if config.kube_config.use_acicni_priority_class %}
+{% if config.flavor != "openshift-4.9-openstack" %}
```

Comment:
We need to do this in a way such that it works for 4.9 and above. You are probably better off checking for equality match with flavors prior to 4.9.

Reply:
sure. good point. Let me correct this